### DISCUSSION TITLE
how to query contentitems under the specified taxonomy

### DISCUSSION THREAD

**Discussion by hyihao** [OrchardCMS/OrchardCore#12304] [Q&A]
Is there a simple case to show (query) the content items under the specified taxonomy?
and how to display the previous and next items of a contentitem

**Comment by MikeAlhayek** [comment]
This should be opened as discussion not as an issue. 

But you should be able to do something like this 

```
await _session.Query<ContentItem, TaxonomyIndex>(x => x.Published && x.TermContentItemId == contentItemId).ListAsync();
```

Can some convert this to discussion?
